app/src/vanilla/java/cm/aptoide/pt/app/view/EditorialItemsViewHolder.java

```diff
@@ -133,15 +134,8 @@ private void manageMediaVisibility(EditorialContent editorialItem) {
           image.setVisibility(View.VISIBLE);
         }
         if (editorialMedia.isVideo()) {
-          if (editorialMedia.getThumbnail() != null) {
-            ImageLoader.with(itemView.getContext())
-                .load(editorialMedia.getThumbnail(), videoThumbnail);
-          }
-          if (editorialMedia.hasUrl()) {
-            videoThumbnailContainer.setVisibility(View.VISIBLE);
-            videoThumbnailContainer.setOnClickListener(
-                v -> editorialMediaClicked.onNext(editorialMedia.getUrl()));
-          }
+          videoThumbnailContainer.setVisibility(View.VISIBLE);
+          setupVideo(editorialMedia.getUrl());
```

Comment:
Isn’t it better to check if the request came with a Url?

Reply:
fixed